client/ayon_applications/utils.py

```diff
@@ -28,6 +28,19 @@
 from .exceptions import MissingRequiredKey, ApplicationLaunchFailed
 from .manager import ApplicationManager
 
+# NOTE Remove try -> except block, when ayon-core > 1.1.1 is required,
+#   or require the version to remove it.
+try:
+    # Was introduced in ayon-core 1.1.2
+    from ayon_core.lib import (
+        merge_env_variables,
+        compute_env_variables_structure,
+    )
+except ImportError:
+    merge_env_variables = None
+    compute_env_variables_structure = None
```

Comment:
Why not move the "legacy function definitions" here?
So that in the code we're always using the new naming already - and at some point we'd just need to remove the `ImportError` fallback?

So in the except case we'd just do:

```python
def merge_env_variables(env, current_env):
    """Modified function(merge) from acre module."""
    result = current_env.copy()
    for key, value in env.items():
        # Keep missing keys by not filling `missing` kwarg
        value = acre.lib.partial_format(value, data=current_env)
        result[key] = value
    return result

def compute_env_variables_structure(merged_env):
    return acre.compute(merged_env, cleanup=False)
```

Reply:
Ohh, lovely, I liek it :).

Reply:
Done